frigate/http.py

```diff
@@ -834,6 +834,22 @@ def label_thumbnail(camera_name, label):
         response.headers["Cache-Control"] = "no-store"
         return response
 
+@bp.route("/<camera_name>/<label>/last_clip.mp4")
+def label_last_clip(camera_name, label):
+    label = unquote(label)
+    event_query = Event.select(fn.MAX(Event.id)).where(Event.camera == camera_name, Event.has_clip == True)
+    if label != "any":
+        event_query = event_query.where(Event.label == label)
+
+    try:
+        event = event_query.scalar()
```

Comment:
```suggestion
        event = event_query.get()
```

Reply:
event ids are a string so scalar won't return correctly

Reply:
I used it like here:
https://github.com/blakeblackshear/frigate/blob/f6a4c2a7b3482acd9c787edf6dbb51898a1c021a/frigate/http.py#L825

Should this line than also be changed?

Reply:
hmm, guess it's fine, I find it odd that that one works